stubs/symfony.php

```diff
@@ -29,6 +29,10 @@
     | mind, that all added `Insights` must belong to a specific `Metric`.
     |
     */
+    
+    'exclude' => [
+        //  'DirectoryOrFileToExclude'
```

Comment:
```suggestion
        //  'path/to/directory-or-file'
```

Reply:
Note: If something exists on symfony by default, maybe you should ignore that folder on the symfony preset: https://github.com/nunomaduro/phpinsights/blob/master/src/Application/Adapters/Symfony/Preset.php

Reply:
I would not add that by default, just because some people might want to include that directory as-is, so imho cleaner to just provide information how to exclude some directories.